app/helpers/ZenifyFixtures/DI/FixturesExtension.php

```diff
@@ -41,35 +39,19 @@ public function beforeCompile()
         $containerBuilder = $this->getContainerBuilder();
         $containerBuilder->resolve();
 
-        $this->loadFakerProvidersToAliceLoader();
-        $this->loadParsersToAliceLoader();
+        $this->loadFakerConfiguration();
     }
 
 
-    private function loadFakerProvidersToAliceLoader()
+    private function loadFakerConfiguration()
```

Comment:
Faker?

Reply:
Faker is the name of the library which is used for data generation.
See https://github.com/FakerPHP/Faker